package.json

```diff
@@ -15,7 +15,10 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start:silent": "BROWSER=none react-scripts start",
```

Comment:
You need to set the PORT when you start the server, otherwise you'll start the default (3000)